### PR TITLE
[ContactStructuralMechanicsApplication] Minor update of utilities

### PR DESCRIPTION
**Pull request by loumalouomega**
Just that, updating tests

### CODE REVIEW

**Review by AlejandroCornejo — DISMISSED** [review]

**Comment by loumalouomega** [comment]
Sorry @AlejandroCornejo I updated something

File: applications/ContactStructuralMechanicsApplication/custom_utilities/mortar_explicit_contribution_utilities.cpp [hunk 1]
```diff
@@ -297,7 +297,7 @@ typename MortarExplicitContributionUtilities<TDim,TNumNodes,TFrictional, TNormal
             const BoundedMatrix<double, TNumNodesMaster, TDim> delta_x2 = x2 - MortarUtilities::GetCoordinates<TDim,TNumNodesMaster>(r_master_geometry, false, 1);
 
             // Delta non-objetive gap and slip
-            noalias(slip_time_derivative)  = - (prod(DOperator, delta_x1) - prod(MOperator, delta_x2))/delta_time;
```
Comment (AlejandroCornejo) [inline applications/ContactStructuralMechanicsApplication/custom_utilities/mortar_explicit_contribution_utilities.cpp]: oops jeje

File: applications/ContactStructuralMechanicsApplication/tests/cpp_tests/test_weighted_gap.cpp [hunk 2]
```diff
@@ -43,6 +43,7 @@ namespace Kratos
 //             gid_io.WriteNodalResults(WEIGHTED_SLIP, rModelPart.Nodes(), label, 0);
 //             gid_io.WriteNodalResultsNonHistorical(NORMAL_GAP, rModelPart.Nodes(), label);
 //             gid_io.WriteNodalResultsNonHistorical(NODAL_AREA, rModelPart.Nodes(), label);
+//             gid_io.WriteNodalResultsNonHistorical(NODAL_VOLUME, rModelPart.Nodes(), label);
```
Comment (AlejandroCornejo) [inline applications/ContactStructuralMechanicsApplication/tests/cpp_tests/test_weighted_gap.cpp:46]: do you want to leave this there?

**Review by AlejandroCornejo — APPROVED** [review]

File: applications/ContactStructuralMechanicsApplication/tests/cpp_tests/test_weighted_gap.cpp (hunk 2 above)
Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/tests/cpp_tests/test_weighted_gap.cpp:46]: Yes, it is useful for debugging

File: applications/ContactStructuralMechanicsApplication/custom_utilities/mortar_explicit_contribution_utilities.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/custom_utilities/mortar_explicit_contribution_utilities.cpp]: This has been a pain in the ass

**Pull request merged**